HTTPServerProject.Tests/HTTPServerWriteTests/TestWriteStreams.cs

```diff
@@ -0,0 +1,46 @@
+using HTTPServerWrite.Streams;
+
+namespace HTTPServerWriteTests.Streams
+{
+    public class TestWriteStreams : IWriteStreams
+    {
+        List<string> sArr = new List<string>();
+
+        public TestWriteStreams(List<string> arr)
+        {
+            sArr = arr;
+        }
+
+        public void Write(string str = default!)
+        {
+            if (str == default!)
+            {
+                sArr.Add("");
+            }
+            else
+            {
+                sArr.Add(str);
+            }
+        }
+
+        public void WriteLine(string str = default!)
+        {
+            if (str == default!)
+            {
+                sArr.Add("");
+            }
+            else
+            {
+                sArr.Add(str);
+            }
+        }
+
+        public void Flush()
```

Comment:
I'm surprised these don't have booleans set in them - you could use these to test if something is flushed and closed.

Reply:
Agreed.